doc/source/whatsnew/v1.2.0.rst

```diff
@@ -143,6 +143,7 @@ See :ref:`install.dependencies` and :ref:`install.optional_dependencies` for mor
 Deprecations
 ~~~~~~~~~~~~
 - Deprecated parameter ``inplace`` in :meth:`MultiIndex.set_codes` and :meth:`MultiIndex.set_levels` (:issue:`35626`)
+- :func:`read_excel` "xlrd" engine is deprecated for all file types that can be handled by "openpyxl" because "xlrd" is no longer maintained (:issue:`28547`).
```

Comment:
can you reword for the benefit end users.

i.e. the default engine for `read_excel` is changing the the future

maybe say something like openpyxl is the recommended engine as xlrd is no longer maintained

Reply:
done